scripts/chicago_isac_scraper.py

```diff
@@ -438,31 +592,74 @@ def cleaningText(metaData):
     for i, dataDict in enumerate(metaData):
         for key, value in dataDict.items():
             if type(value) == str:
-                metaData[i][key] = re.sub(r'[\n\t\\\"]', '', metaData[i][key])
-                metaData[i][key] = re.sub(r'[\u00a0]', ' ', metaData[i][key])
+                isbnPattern = re.compile(r'ISBN\-10 |ISBN\-10|ISBN\-13 |ISBN\-13|ISBN |ISBN|\:|\
+                                         \(hardback\) | \(hardcover\) |\(eBook\) ')
+                charPattern = re.compile(r'\n|\t|\\|\\:|\u00a0')
+                authorPattern = re.compile(r' Originally published in ([\d]+).')
+                titlePattern = re.compile(r'AS [\d]+. |CHDS [\d]+. |ISAC [\d]+. | OIS [\d]+. |LAMINE [\d]+. |\
+                                          MAD [\d]+. |MSKH [\d] |NE [\d]+. |OIC [\d]+. |OIDA [\d]+. |OIMP [\d]+. |\
+                                          OIP [\d]+. |SAOC [/d]+. ')
+                metaData[i][key] = re.sub(charPattern, '', metaData[i][key])
+                metaData[i][key] = re.sub(isbnPattern, '', metaData[i][key])
+                metaData[i][key] = re.sub(authorPattern, '', metaData[i][key])
+                metaData[i][key] = re.sub(titlePattern, '', metaData[i][key])
+                metaData[i][key] = metaData[i][key].strip()
+    
     return metaData
 
+def cleaningAuthorMetadata(authors):

```

Comment:
One other thing that's not too urgent in a script but more a nit: it can be easier to read function names in a more imperative (? not sure if that's the right word lol) tone, ie, something like `cleanAuthorMetadata`